datapreview.tex

```diff
@@ -58,69 +52,42 @@ \subsubsection{Data Preview 0.2}
 \subsubsection{Data Preview 0.3}
 \label{sec:dp03}
 
-Scheduled for between June and September 2023, DP0.3 will be the last in the DP0 series of Data Previews based on simulated LSST-like data. 
-The main goal of DP0.3 is to support the Solar System Science Collaboration by hosting their simulated 10-year catalog and serving it via the RSP at the US DAC. 
-Table~\ref{tab:dp-zero-three-products} presents a summary of the expected DP0.3 data products.
-The exact data products for DP0.3 are still to be decided. 
+Delivered in June  2023, DP0.3 is the last in the DP0 series of Data Previews based on simulated LSST-like data. 
+DP0.3 supports the Solar System Science Collaboration by hosting their simulated 1-year and 10-year Prompt Products Database )(PPDB) catalog to enable moving object analysis development  in  RSP at the US DAC. 
+
+For full details on DP0.3 including an exact description of the data products served, see the documentation at \url{https://dp0-3.lsst.io/}
 
 \subsection{Data Preview 1}
 \label{sec:dp1}
 
-Data Preview 1 was originally defined to be based on reprocessed on-sky data taken with ComCam.
-Following the replan of the Construction project in December 2022, no on-sky data will be taken with ComCam, \S~\ref{sec:commissioning}.
-Consequently DP1 has been redefined to be based on a subset of science-grade images taken with LSSTCam during a period of a few days around the System First Light milestone, \S~\ref{ssec:commissioning-observations}.
-The processing pipelines and exact data products that will comprise DP1 are still under discussion. 
-At minimum, DP1 will deliver visit-level images and catalogs to enable initial studies of observational and instrumental effects. 
+Data Preview 1 (DP1) will serve data products generated from a subset of science-grade astronomically-useful images as soon as 
+possible following the end of either ComCam on sky or System First Light (\S~\ref{ssec:commissioning-milestones}) to enable the community to prepare to work with LSST data. 
+In the current schedule, which re-implements on-sky data-taking with ComCam for 2 months,  DP1 would be based on 2 months of ComCam data. 
+Note that the time period during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken by the end of ComCam work on sky or around System First Light.
 
-Note that the DP1 period of time during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken in the few days around System First Light.
-Table~\ref{tab:dp-one-products} presents a summary of the data products expected in DP1.
+In both DP1 scenarios, the exact processing pipelines and data products are still under discussion.
+At minimum,  DP1  will deliver visit-level images and catalogs to enable initial studies of observational and instrumental effects. 
```

Comment:
Do you want to say "calibrated" or "provisionally calibrated" images?

Reply:
Have we defined "provisionally calibrated" anywhere? All the data products in the DP program will be provisional, in that they are produced with in-development versions of the pipelines